src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,37 @@
-export const Tabs = () => {};
+import React from 'react';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
+
+  return (
+    <div>
```

Comment:
```suggestion
    <div data-cy="TabsComponent">
```
And remove this div wrapper in App.jsx